src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -2091,6 +2091,7 @@ End Module").Path
             InlineData("15.3", True, LanguageVersion.VisualBasic15_3),
             InlineData("15.5", True, LanguageVersion.VisualBasic15_5),
             InlineData("16", True, LanguageVersion.VisualBasic16),
+            InlineData("16.0", True, LanguageVersion.VisualBasic16),
```

Comment:
@jcouv Would it make more sense if we change:

```
        Public Sub LanguageVersion_TryParseDisplayString(input As String, success As Boolean, expected As LanguageVersion)
            Dim version As LanguageVersion
            Assert.Equal(success, TryParse(input, version))
            Assert.Equal(expected, version)
            ' The canary check is a reminder that this test needs to be updated when a language version is added
        End Sub
```

to:

```
        Public Sub LanguageVersion_TryParseDisplayString(input As String, success As Boolean, expected As LanguageVersion)
            Dim version As LanguageVersion
            Assert.Equal(success, TryParse(input, version))
            Assert.Equal(expected, version)

            If input.IsAllDigits() Then// IsAllDigits is just a pseudo code.
                Dim version2 As LanguageVersion
                Assert.Equal(success, TryParse(input & ".0", version2))
                Assert.Equal(expected, version2)
            End If

            ' The canary check is a reminder that this test needs to be updated when a language version is added
        End Sub
```

Reply:
Sure

Reply:
I think I'd prefer if we keep the explicit tests for the things that we expect to work. It makes the test simpler and less magical.